CrackingTheCodingInterview/Chapter7MathematicsAndProbability/Operations.java

```diff
@@ -40,6 +40,9 @@ public static int abs(int a) {
 		else {
 			return a;
 		}
+		
+		//or
+		return Math.abs(a);
```

Comment:
No problem! Hopefully you can contribute in some other way. We're always looking to improve this repository.